app/assets/scripts/views/faq.js

```diff
@@ -0,0 +1,145 @@
+              <p>Những mã VPRoMMS đã được tải lên được hiển thị màu đỏ.</p>
+            </section>
+        }
+        {
+          language === 'en' ?
+            <section className="question">
+              <h3 className='inpage__title'>Why can’t I click on some VPRoMMS IDs?</h3>
+              <p>You can only choose a VPRoMMS ID when it has data attached to (the red one)</p>
+            </section> :
+            <section className="question">
+              <h3 className='inpage__title'>Tại sao tôi không thể chọn một số mã VPRoMMS?</h3>
+              <p>Bạn chỉ có thể chọn một mã VPRoMMS khi nó có dữ liệu đính kèm (mã màu đỏ)</p>
+            </section>
+        }
+        {
+          language === 'en' ?
+            <section className="question">
+              <h3 className='inpage__title'>Why are there two different values of road length in one VPRoMMS ID?</h3>
+              <p>There are two values of length in a VPRoMMS ID: one comes from VPRoMMS speardsheet, one is calculated from GPS trace in the field. In some cases, these can be different.</p>
+            </section> :
+            <section className="question">
+              <h3 className='inpage__title'>Tại sao có hai giá trị của chiều dài đường trong một mã VPRoMMS?</h3>
+              <p>Có hai giá trị của chiều dài đường trong một mã VPRoMMS: một là từ bảng dữ liệu VPRoMMS, một là được tính từ dữ liệu GPS từ hiện trường. Trong một số trường hợp, những dữ liệu này có thể khác nhau.</p>
+            </section>
+        }
+        {
+          language === 'en' ?
+            <section className="question">
+              <h3 className='inpage__title'>What is ‘road without VPRoMMS ID’?</h3>
+              <p>Some previously uploaded roads have no VPRoMMS IDs or the VPRoMMS IDs mismatch with VPRoMMS ID list in ORMA.</p>
+            </section> :
+            <section className="question">
+              <h3 className='inpage__title'>‘đường không có mã VPRoMMS’ là gì?</h3>
+              <p>Một số đường đã được tải lên từ trước không có mã VPRoMMS hoặc mã VPRoMMS không khớp với danh sách mã VPRoMMS trong ORMA</p>
+            </section>
+        }
+        {
+          language === 'en' ?
+            <section className="question">
+              <h3 className='inpage__title'>In ‘Tasks’, the system shows that there are 3 roads but I can only see 2 roads.</h3>
+              <p>Please try to zoom in and look around the roads, you may find the third one.</p>
+            </section> :
+            <section className="question">
+              <h3 className='inpage__title'>Trong mục ‘Nhiệm vụ’, hệ thống báo rằng có 3 đường đang được hiển thị nhưng tôi chỉ thấy 2 đường.</h3>
+              <p>Hãy thử phóng to và tìm xung quanh các đường đang hiển thị, bạn có thể tìm thấy đường thứ ba.</p>
+            </section>
+        }
+
+        <section>
+          <form action="mailto:dphan2@worldbank.org" method="GET">
```

Comment:
In the `mailto` link, can we also include an email subject, that updates with the language toggle to boot?